plugins/lyrics-genius/back.js

```diff
@@ -7,20 +7,46 @@ const fetch = require("node-fetch");
 
 const { cleanupName } = require("../../providers/song-info");
 const { injectCSS } = require("../utils");
+var revRomanized = false; 
```

Comment:
use `let` instead of `var`

```suggestion
let revRomanized = false; 
```